inference-engine/tests/functional/inference_engine/lp_transformations/clamp_transformation.cpp

```diff
@@ -0,0 +1,102 @@
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+#include "simple_low_precision_transformer.hpp"
+#include "ngraph_functions/low_precision_transformations/clamp_function.hpp"
+
+
+using namespace testing;
+using namespace ngraph::pass;
+
+class ClampTransformationTestValues {
+public:
+    low_precision::LayerTransformation::Params params;
+    std::vector<float> subtractValues;
+    std::vector<float> multiplyValues;
+};
+
+typedef std::tuple<
+    ngraph::element::Type,
+    ngraph::Shape,
+    ClampTransformationTestValues> ClampTransformationParams;
+
+class ClampTransformation : public LayerTransformation, public testing::WithParamInterface<ClampTransformationParams> {
+public:
+    void SetUp() override {
+        const ngraph::element::Type precision = std::get<0>(GetParam());
+        const ngraph::Shape shape = std::get<1>(GetParam());
+        const ClampTransformationTestValues testValues = std::get<2>(GetParam());
+
+        actualFunction = ngraph::builder::subgraph::ClampFunction::getOriginal(precision,
+            shape,
+            {
+                testValues.params.updatePrecisions ? testValues.params.precisionsOnActivations[0] : precision,
+                testValues.subtractValues,
+                testValues.multiplyValues
+            });
+
+        SimpleLowPrecisionTransformer transform;
+        transform.add<ngraph::pass::low_precision::ClampTransformation, ngraph::opset1::Clamp>(testValues.params);
+        transform.transform(actualFunction);
+
+        referenceFunction = ngraph::builder::subgraph::ClampFunction::getReference(precision,
+            shape,
+            {
+                testValues.params.updatePrecisions ? testValues.params.precisionsOnActivations[0] : precision,
+                testValues.subtractValues,
+                testValues.multiplyValues
+            });
+    }
+
+    static std::string getTestCaseName(testing::TestParamInfo<ClampTransformationParams> obj) {
+        const ngraph::element::Type precision = std::get<0>(obj.param);
+        const ngraph::Shape shape = std::get<1>(obj.param);
+        const ClampTransformationTestValues testValues = std::get<2>(obj.param);
+        return LayerTransformation::getTestCaseNameByParams(precision, shape, testValues.params);
+    }
+};
+
+TEST_P(ClampTransformation, CompareFunctions) {
+    InitNodeInfo().run_on_function(actualFunction);
+    actualFunction->validate_nodes_and_infer_types();
+
+    auto res = compare_functions(referenceFunction, actualFunction, true);
+    ASSERT_TRUE(res.first) << res.second;
+}
+
+const std::vector<ngraph::element::Type> precisions = {
+    ngraph::element::f32,
+    ngraph::element::f16
+};
+
+const std::vector<ngraph::Shape> shapes = {
+    { 1, 32, 72, 48 },
+    { 1, 3, 224, 224 }
+};
+
+const std::vector<ClampTransformationTestValues> testValues = {
+    { LayerTransformation::createParamsU8I8(), { 128 }, { 3.0f } },
+    { LayerTransformation::createParamsI8I8(), { 2 }, { -5.0f } }
+};
```

Comment:
can you add additional test cases:
1. per-channel quantization
2. updatePrecision = false (no INT8 tensors in model)

Reply:
Done, thanks!